controllers/component_build_controller_test.go

```diff
@@ -1288,6 +1413,20 @@ var _ = Describe("Component initial build controller", func() {
 			ensureNoPipelineRunsCreated(resourceKey)
 		})
 
+		It("should not submit initial build if pac secret is missing", func() {
+			deleteSecret(pacSecretKey)
+			setComponentDevfileModel(resourceKey)
+			ensureNoPipelineRunsCreated(resourceKey)
```

Comment:
My concern with `ensure*` methods is that they use `Consistently` under the hood (unlike `wait*` that use `Eventually`), which means each invocation takes `ensureTimeout` i.e. `4` seconds.
I don't mind they're used, but would like to do it reasonable to keep total testing time minimal (but not sacrificing quality, of course).

Reply:
ok I'll check where I can avoid it